cmd/flux/tree.go

```diff
@@ -23,7 +23,7 @@ import (
 var treeCmd = &cobra.Command{
 	Use:   "tree",
 	Short: "Print the resources reconciled by Flux",
-	Long:  `The tree command shows the list of resources reconciled by a Flux object.'`,
+	Long:  withPreviewNote(`The tree command shows the list of resources reconciled by a Flux object.`),
```

Comment:
`tree` expands HelmReleases, let's keep tree and logs with the warning for now.

Reply:
i marked it as experimental, because `tree` is not a trivial command like `suspend`, `resume`, etc. and we might add support for other kinds such as HelmReleases. but as per your later comment, since we are not marking command groups as experimental and only the actual commands/subcommands let me remove the note from here.

Reply:
okay @stefanprodan